recipes/capstone/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "capstone" %}
+{% set version = "4.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/aquynh/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 79bbea8dbe466bd7d051e037db5961fdb34f67c9fac5c3471dd105cfb1e05dc7
+
+build:
+  number: 0
+  skip: True  # [win or osx or py < 30]
```

Comment:
Do those spaces work? I thought it had to be py<30.

Reply:
nope! they work, it is just Python!